Update README, fix HLint CI (#417)
kowainik/stan: README.md

more guarantees from your code, not provided by the Haskell type
system or GHC.

For a crash course to Stan, watch the talk about Stan, presented by
Veronika Romashkina and Dmitrii Kovanikov at the Haskell Love
conference.

> * [Stan – Haskell Static Analyser](https://www.youtube.com/watch?v=wb5PLv6-e6I)

Reviewer: Do you think that embedding the vide would be a good solution?

```suggestion
[![Stan – Haskell Static Analyser](https://img.youtube.com/vi/wb5PLv6-e6I/0.jpg)](https://www.youtube.com/watch?v=wb5PLv6-e6I)
```

We have much more ideas to work on.
See more detailed plan in the dedicated [GitHub Project page](https://github.com/kowainik/stan/projects/1).

## Users

> Stan is known to be used in the following companies:

Reviewer: Alternative: adopted by, integrated to the companies' processes etc.?

Author: I like "Adopted by" 👍🏻 Does the following look okay?

```suggestion
Stan is known to be adopted by the following companies:
```
